typings/index.d.ts

```diff
@@ -0,0 +1,76 @@
+declare module 'ytdl-core' {
+  import { ClientRequest } from 'http';
+  import { Readable } from 'stream';
+
+  export function ytdl(link: string, options?: DownloadOptions): Readable;
+  export namespace ytdl {
+    export function getInfo(url: string, callback?: (err: Error, info: VideoInfo) => void): Promise<VideoInfo>;
+    export function getInfo(url: string, options?: DownloadOptions, callback?: (err: Error, info: VideoInfo) => void): Promise<VideoInfo>;
+    export function downloadFromInfo(info: VideoInfo, options?: DownloadOptions): Readable;
+    export function chooseFormat(format: object[], options?: DownloadOptions): object | Error;
+    export function filterFormats(formats: object[], filter?: 'video' | 'videoonly' | 'audio' | 'audioonly' | ((format: VideoFormat) => boolean)): object[];
+  }
+
+  export type VideoFormat = {
+    fallback_host: string;
+    quality: string;
+    type: string;
+    url: string;
+    itag: string;
+    sig: string;
+    container: 'flv' | '3gp' | 'mp4' | 'webm' | 'ts';
+    resolution: '144p' | '240p' | '270p' | '360p' | '480p' | '720p' | '1080p' | '1440p' | '2160p' | '3072p';
+    encoding: 'Sorenson H.283' | 'MPEG-4 Visual' | 'VP8' | 'VP9' | 'H.264';
+    profile: '3d' | 'high' | 'main' | 'simple' | 'baseline' | 'Main@L3.1';
+    bitrate: string;
+    audioEncoding: 'mp3' | 'vorbis' | 'aac' | 'opus' | 'flac';
+    audioBitrate: number;
+  }
+
+  export type VideoInfo = {
+    plid: string;
+    sendtmp: number;
+    title: string;
+    status: string;
+    muted: number;
+    pltype: string;
+    thumbnail_url: string;
+    has_cc: boolean;
+    video_verticals: number[];
+    track_embed: number;
+    author: string;
```

Comment:
There's been some changes since the last PR, `author` is now an object that looks like [this](https://github.com/fent/node-ytdl-core/blob/master/example/info.json#L51)

And it also now includes [`published` and `relatedVideos`](https://github.com/fent/node-ytdl-core/blob/master/example/info.json#L620)